src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Order.xml

```diff
@@ -44,12 +39,13 @@
             <group>sylius:admin:order:index</group>
             <group>sylius:admin:order:show</group>
             <group>sylius:shop:order:index</group>
-            <group>sylius:shop:order:show</group>
+            <group>sylius:shop:cart:show</group>
         </attribute>
 
         <attribute name="customer">
             <group>sylius:admin:order:index</group>
             <group>sylius:admin:order:show</group>
+            <group>sylius:shop:cart:show</group>
```

Comment:
We should add CustomerExtension or CustomerProvider for shop section.

Reply:
We chose to embed the `[customer][email]` field here, or show `null` if no customer is set.